src/TestUtils/src/Microsoft.Maui.IntegrationTests/TemplateTests.cs

```diff
@@ -181,6 +182,36 @@ public void BuildWithDifferentVersionNumber(string id, string config, string dis
 				$"Project {Path.GetFileName(projectFile)} failed to build. Check test output/attachments for errors.");
 		}
 
+		[Test]
+		[TestCase("maui-blazor", "Debug", "net8.0")]
+		[TestCase("maui-blazor", "Release", "net8.0")]
```

Comment:
main is still net7. I'll do a separate PR to fix this so we never have to do this again.

```suggestion
		[TestCase("maui-blazor", "Debug", "net7.0")]
		[TestCase("maui-blazor", "Release", "net7.0")]
```

Reply:
https://github.com/dotnet/maui/pull/17336